### ISSUE TITLE
Classnames starting with number cause syntax error

### ISSUE THREAD

**Issue by m0rtalis** [Jimdo/typings-for-css-modules-loader#73]
Classnames starting with a number [0-9] cause a syntax error in the generated .css.d.ts file.

An example is the classname [`35 00px` in SemanticUI](https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/semantic.css#L8286) which results in the type definition `export const 35: string;`,  which obviously throws syntax errors.

A fix is to change the [allWordsRegexp](https://github.com/Jimdo/typings-for-css-modules-loader/blob/master/src/cssModuleToInterface.js#L21) to `/^[a-zA-Z]\w*$/i`. Although no idea if that's the best approach.

**Comment by SaphuA** [comment]
Interesting.. because css officially doesn't allow digits as the first character of a class name.

**Comment by m0rtalis** [comment]
Yes apparently they have efforts to change the class name Semantic-Org/Semantic-UI#5468 but in the meanwhile can the regex be changed to support such class names?
If yes I can create a PR for this
